plugins/prebuffer-mixin/src/main.ts

```diff
@@ -591,6 +591,9 @@ class PrebufferSession {
         //   ffmpegInput.inputArguments.unshift('-use_wallclock_as_timestamps', '1');
 
         // extraOutputArguments must contain full codec information
+        if (ffmpegInput.h264EncoderArguments) {
```

Comment:
move this up a line to preserve comment meant for below.

Reply:
done